public/locales/en-US.properties

```diff
@@ -56,3 +58,6 @@ create-new-wallet = Create new
 wallet-mnemonic = Mnemonic
 wallet-seed = Seed
 wallet-password = Set a password for your wallet
+
+# Soft keys
+softkey-capture-camera = Capture QR
```

Comment:
`19:51:33.725 "L10nError: "actions" not found in en-US in app://7324e2ef-f958-44d3-a836-1f439d4258f8/index.html"1bundle.js:53:62125`
I guess actions is missing here.